client/src/ApiTest/testApi.mjs

```diff
@@ -0,0 +1,71 @@
+// testAuthServiceWithAPI.js
+import axios from 'axios';
+
+// Mock sessionStorage for testing purposes
+global.sessionStorage = {
+  store: {},
+  getItem(key) {
+    return this.store[key] || null;
+  },
+  setItem(key, value) {
+    this.store[key] = value;
+  },
+  removeItem(key) {
+    delete this.store[key];
+  }
+};
+
+// Set up actual publicApi using Axios for real API requests
+const publicApi = axios.create({
+  baseURL: 'http://127.0.0.1:5001/api/v1',  
+  withCredentials: true  // Include cookies with requests
+
+});
+
+// authService with real publicApi
+const authService = {
+  getToken: () => sessionStorage.getItem('authToken'),
+  setToken: (token) => sessionStorage.setItem('authToken', token),
+  removeToken: () => sessionStorage.removeItem('authToken'),
+  
+
+  refreshAccessToken: async () => {
+    try {
+      // Make a request to refresh the access token
+      const response = await publicApi.post(
+        '/auth/refresh',
+        {},
+        {
+          headers: {
+            Cookie: 'refresh_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJmcmVzaCI6ZmFsc2UsImlhdCI6MTczMDU1NDI0MCwianRpIjoiNWExM2FiMDUtMjM1Ny00NDFiLWFhODgtMzAzODA0NjFlYzQzIiwidHlwZSI6InJlZnJlc2giLCJzdWIiOjEsIm5iZiI6MTczMDU1NDI0MCwiY3NyZiI6IjI4MDUwOTdhLWE5MzItNGQxNi05OTc0LTU5MjUyZDE3OTQ2NCIsImV4cCI6MTczMTE1OTA0MH0.VoaaWYFwi7gAVrFKyvPu7CvQscmppESubVV5Ef07YpM'
```

Comment:
what is this?

Reply:
For testing purposes only, I want it to be accessible on all devices.

Reply:
is it a valid token? won't it expires?

Reply:
Just like curl.txt examples